src/core/qfieldcloud/qfieldcloudproject.cpp

```diff
@@ -1064,22 +1087,19 @@ void QFieldCloudProject::downloadFiles()
       continue;
     }
 
-    NetworkReply *reply = downloadFile( mDownloadFileTransfers[fileKey].projectId, mDownloadFileTransfers[fileKey].fileName, mDownloadFileTransfers[fileKey].projectId == mId );
+    FileTransfer &transfer = mDownloadFileTransfers[fileKey];
```

Comment:
```suggestion
    FileTransfer &fileTransfer = mDownloadFileTransfers[fileKey];
```

Slight preference to explicitly name the object. No super strong opinion.

Also, can it be a const?